### ISSUE TITLE
Erratic cursor movement

### ISSUE THREAD

**Issue by bitcraft** [Tuxemon/Tuxemon#770]
Please do not open any new issues or PRs for the menu cursor unless it is so broken that it cannot select items.  There are currently known issues with details, but it should *work*.  I don't want any fixes now until there is test coverage on the cursor code.

**Comment by kerizane** [comment]
Recent bugs relating to cursor:
https://github.com/Tuxemon/Tuxemon/issues/735
https://github.com/Tuxemon/Tuxemon/issues/718
https://github.com/Tuxemon/Tuxemon/issues/714
https://github.com/Tuxemon/Tuxemon/issues/698
https://github.com/Tuxemon/Tuxemon/issues/728

Should try to get tests against these.

**Comment by kerizane** [comment]
Still no tests for this, right?

**Comment by kerizane** [comment]
New issue:

Game crashes if you sell the last item in a list (but not if it's the only item).
